python/lsst/ip/isr/isrQa.py

```diff
@@ -27,6 +27,9 @@
 import lsst.pex.config as pexConfig
 
 
+__all__ = ['IsrQaFlatnessConfig', 'IsrQaConfig', 'makeThumbnail']
```

Comment:
This one is also duplicating one that is higher up the file. I'll stop commenting about them now since you should be able to find ones that have multiple `__all__` in a single file.

Reply:
It was all done in #227.